style.css

```diff
@@ -0,0 +1,790 @@
+@font-face {
+  font-family: "Robot";
+  src: url(./Assets/Fonts/RobotoCondensed-Regular.ttf);
+}
+
+html,
+body {
+  height: 100%;
+  overflow: auto;
+}
+
+body {
+  background-color: #f4f5f6 !important;
```

Comment:
Use of `!important` is considered a bad practice, please remove it from your project.